assets/css/tasks/_contact-tasks.css

```diff
@@ -0,0 +1,70 @@
+/* CONTACT PAGE */
+
+/* assets/css/pages/_contact.css */
+.contact-section {
+    background-image: url(/img/contact-img.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: right 20rem bottom 0;
+    
+}
+
+.form {
+    padding: 10px 10px;
```

Comment:
If both directions have the same value, you can set just one value. In this case `padding: 10px`;